controller_report_xls/controllers/main.py

```diff
@@ -38,6 +38,96 @@
 _logger = logging.getLogger(__name__)
 
 
+def is_float(value):
+    try:
+        float(value)
+        return True
+    except ValueError:
+        return False
+
+
+def is_integer(value):
+    try:
+        int(value)
+        return True
+    except ValueError:
+        return False
```

Comment:
it is always

isinstance(value, (int, float)) => False

because:

type(value) => string.

Regards